### PR TITLE
Fix tiny typo in docs for chi2 pdf

### PR DESCRIPTION
**Pull request by finefoot**
I was reading the docs at https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.chi2.html and this tiny typo in the math formula caught my eye. Hopefully, it's alright to propose such tiny pull request.

### CODE REVIEW

**Comment by ev-br** [comment]
Thanks! It is indeed most welcome.

BTW, There's one more typo in this line, the `gamma` function should be uppercase `\Gamma`. Care to fix it too?
You can just push a second commit to the same branch

**Comment by WarrenWeckesser** [comment]
Thanks @emmi474.  This was also pointed out on the mailing list back in April: https://mail.python.org/pipermail/scipy-dev/2018-April/022752.html

**Comment by emmi474** [comment]
@WarrenWeckesser 
>  This was also pointed out on the mailing list back in April: https://mail.python.org/pipermail/scipy-dev/2018-April/022752.html

Sorry, of course. Thank you. I had a look at the issues and pull requests here on GitHub but didn't think of having a look at the mailing list archives.

@ev-br 
> BTW, There's one more typo in this line, the gamma function should be uppercase \Gamma. Care to fix it too?

Done for `chi` and `chi2`. However, there seem to be 16 other `\gamma` and I don't know which ones have to be `\Gamma`.

Furthermore, I took the liberty to make the math formulas for the `chi` and `chi2` probability density functions "look more comparable". I hope that's desirable?

**Comment by emmi474** [comment]
Looking at https://github.com/scipy/scipy/pull/8865/commits/96ed8649e3aafd004beff505473a681974d932da I saw chi2 had `for x > 0` instead of ``for :math:`x > 0` `` directly after the formula. Some other lines had the same typo or a similar typo.

What's the desired consistency for the immediate sentence after each formula? I found both
* ``` `chi2` takes ``df`` as a shape parameter.```
* `` `chi2` takes :math:`df` as a shape parameter.``

I would have thought, the first one is better, since `chi2` is referring to the Python function and not the math formula, so `df` should be in monospaced font, not math mode. Both versions exist in different docstrings. Should I change them? And which ones?

By the way, are those continuous integration tests failing because of my changes? Even the 1 character change in the first commit failed... :thinking:

**Comment by rgommers** [comment]
> By the way, are those continuous integration tests failing because of my changes? Even the 1 character change in the first commit failed... 🤔

No, they're unrelated, please just ignore them. I'm opening a separate issue for that now.

**Comment by rgommers** [comment]
I've squashed and rebased this to fix the merge conflict with gh-8874 and to resolve the CI failures.

**Comment by emmi474** [comment]
> Re :math:df etc : I think the best way to write it would be to use :math:\nu in the TeX formula and add a sentence below the formula, along the lines of where :math:\nu is the number of degrees of freedom, denoted as ..double backticks..df..double backticks.. in the code.

@ev-br I've worked on this to make the parameter descriptions a bit more consistent.

Do you want to take a look at it? Or how do I prepare this PR for review?

File: scipy/stats/_continuous_distns.py [hunk 1]
```diff
@@ -6279,13 +6283,15 @@ class gennorm_gen(rv_continuous):
     -----
     The probability density function for `gennorm` is [1]_::
 
-                                     beta
-        gennorm.pdf(x, beta) =  ---------------  exp(-|x|**beta)
-                                2 gamma(1/beta)
+    .. math::
+
+        f(x, \beta) = \frac{\beta}{2 \Gamma(1/\beta)} \exp(-|x|^\beta)
```
Comment (rgommers) [inline scipy/stats/_continuous_distns.py]: This line is responsible for the one test failure:

    scipy.stats.gennorm:30: (WARNING/2) Inline substitution_reference start-string without end-string.::

The problem seems to be `|x|` (I think, but not sure), which was fine before but not now that it's a math block. I don't quite get it though; there are similar math expressions elsewhere and the docstring is raw (`r"""`).

Anyone else spot the problem?

Comment (emmi474) [reply scipy/stats/_continuous_distns.py]: My bad, sorry. I think I fixed it with https://github.com/scipy/scipy/pull/8865/commits/f44bcf67d49dd5260553b51614cde502eadbe174 It was `::` 4 lines above the line which produced the error.

Comment (rgommers) [reply scipy/stats/_continuous_distns.py]: that was it, good catch!

**Comment by rgommers** [comment]
This all looks good now, nice improvement. Thanks @emmi474!

**Pull request merged**